packages/transport/src/modules/relationships/local/Relationship.ts

```diff
@@ -61,6 +61,15 @@ export class Relationship extends CoreSynchronizable implements IRelationship {
     @serialize()
     public metadataModifiedAt?: CoreDate;
 
+    public override toJSON(verbose?: boolean | undefined, serializeAsString?: boolean | undefined): any {
+        const json = super.toJSON(verbose, serializeAsString) as any;
```

Comment:
Why dont we have an type here instead of `any`

Reply:
this is just turning of the compiler for me, so that I can write any data to it

Reply:
I guess he meant using RelationshipJSON as a return value would be better?

Reply:
we don't have JSONs for transport objects, only for contents :)